SConstruct

```diff
@@ -61,6 +62,7 @@ Depends(debrief, BUILD_TARGETS)
 env.AlwaysBuild(debrief)
 if 'state_of_repo.log' not in BUILD_TARGETS: 
     BUILD_TARGETS.append('state_of_repo.log')
+NoCache('state_of_repo.log')
```

Comment:
The comment could be clearer. The subject of this one (the file) is a little hidden. Maybe something like: "Forces log to be rebuilt, won't pull from cache."